packages/browser/package.json

```diff
@@ -22,7 +22,7 @@
     "lint:eslint": "eslint --config .eslintrc.js \"src/\"",
     "lint:prettier": "prettier \"src/*.{ts,tsx,js,jsx,css}\" \"**/*.{md,mdx,yml}\"",
     "licenses:check": "license-checker --production --out license.csv --failOn \"AGPL-1.0-only; AGPL-1.0-or-later; AGPL-3.0-only; AGPL-3.0-or-later; Beerware; CC-BY-NC-1.0; CC-BY-NC-2.0; CC-BY-NC-2.5; CC-BY-NC-3.0; CC-BY-NC-4.0; CC-BY-NC-ND-1.0; CC-BY-NC-ND-2.0; CC-BY-NC-ND-2.5; CC-BY-NC-ND-3.0; CC-BY-NC-ND-4.0; CC-BY-NC-SA-1.0; CC-BY-NC-SA-2.0; CC-BY-NC-SA-2.5; CC-BY-NC-SA-3.0; CC-BY-NC-SA-4.0; CPAL-1.0; EUPL-1.0; EUPL-1.1; EUPL-1.1;  GPL-1.0-only; GPL-1.0-or-later; GPL-2.0-only;  GPL-2.0-or-later; GPL-3.0; GPL-3.0-only; GPL-3.0-or-later; SISSL;  SISSL-1.2; WTFPL\"",
-    "test": "OPENSSL_CONF=/dev/null jest --coverage --verbose",
+    "test": "OPENSSL_CONF=/dev/null jest --coverage --verbose --selectProjects browser --config ../../jest.config.ts",
```

Comment:
Can this be done with a setupFiles setting that mutates process.env? (That is, before the test code is loaded)

Reply:
Interestingly, it can be done in jest's config file, but not as part of the environment setup for some reason: d18fbcb4cf8662ae90410c16b7674b0931e8dc39.